packages/twenty-front/src/modules/workflow/workflow-steps/workflow-actions/code-action/components/WorkflowEditActionServerlessFunction.tsx

```diff
@@ -32,19 +41,82 @@ import { useActionIconColorOrThrow } from '@/workflow/workflow-steps/workflow-ac
 import { getActionIcon } from '@/workflow/workflow-steps/workflow-actions/utils/getActionIcon';
 import { WorkflowVariablePicker } from '@/workflow/workflow-variables/components/WorkflowVariablePicker';
 import styled from '@emotion/styled';
+import { useLingui } from '@lingui/react/macro';
+
 import { Monaco } from '@monaco-editor/react';
 import { editor } from 'monaco-editor';
 import { AutoTypings } from 'monaco-editor-auto-typings';
-import { useEffect, useState } from 'react';
+import { useEffect, useRef, useState } from 'react';
+import { createPortal } from 'react-dom';
 import { useRecoilState } from 'recoil';
+import { Key } from 'ts-key-enum';
 import { isDefined } from 'twenty-shared/utils';
-import { IconCode, IconPlayerPlay, useIcons } from 'twenty-ui/display';
-import { CodeEditor } from 'twenty-ui/input';
+import {
+  IconCode,
+  IconMaximize,
+  IconPlayerPlay,
+  useIcons,
+} from 'twenty-ui/display';
+import { CodeEditor, LightIconButton } from 'twenty-ui/input';
+import { useIsMobile } from 'twenty-ui/utilities';
 import { useDebouncedCallback } from 'use-debounce';
 
+const CODE_EDITOR_MIN_HEIGHT = 343;
+
 const StyledCodeEditorContainer = styled.div`
   display: flex;
   flex-direction: column;
+  position: relative;
+  flex: 1;
+  min-height: ${CODE_EDITOR_MIN_HEIGHT}px;
+  overflow: hidden;
+`;
+
+const StyledFullScreenOverlay = styled.div`
```

Comment:
it looks like this should be part of right drawer tooling, rather than local capability

Reply:
To me it's most likely this would fit in the code editor tooling rather than sidepanel (for example if the code editor is embedded in settings). But we'll see